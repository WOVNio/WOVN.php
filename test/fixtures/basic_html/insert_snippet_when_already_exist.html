<!DOCTYPE html>
<html>
<head><script src="//j.wovn.io:3000/1" data-wovnio="key=4tok3n" async></script><script type="text/javascript" src="fake.js"></script><script src="//j.dev-wovn.io:3000/1" data-wovnio="key=sLz3ob" async></script></head>
<body style="container">
<p>Fake body</p>
</body>
</html>
