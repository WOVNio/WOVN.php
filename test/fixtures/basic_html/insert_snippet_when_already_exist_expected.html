<!DOCTYPE html>
<html>
<head><link rel="alternate" hreflang="en" href="http://ja.localhost/en/t.php?hey=yo"><link rel="alternate" hreflang="vi" href="http://ja.localhost/vi/t.php?hey=yo"><script src="//j.wovn.io/1" data-wovnio="key=toK3n&amp;backend=true&amp;currentLang=ja&amp;defaultLang=ja&amp;urlPattern=path&amp;langCodeAliases=[]&amp;version=WOVN.php" data-wovnio-type="backend_without_api" async></script><script type="text/javascript" src="fake.js"></script></head>
<body style="container">
<p>Fake body</p>
</body>
</html>